tests/testthat/test-ggplot-axis.R

```diff
@@ -0,0 +1,81 @@
+context("Axis moving")
+
+expect_traces <- function(gg, n.traces, name){
+  stopifnot(is.numeric(n.traces))
+  L <- save_outputs(gg, paste0("axis-", name))
+  all.traces <- L$data
+  no.data <- sapply(all.traces, function(tr) {
+    is.null(tr[["x"]]) && is.null(tr[["y"]])
+  })
+  has.data <- all.traces[!no.data]
+  expect_equal(length(has.data), n.traces)
+  list(data = has.data, layout = L$layout)
+}
+
+p <- ggplot(mtcars, aes(x=mpg, y=wt)) + 
+	geom_point()
+
+# p3 <- p + facet_wrap(~carb)
+
+test_that("Axis position moves to top", {
+	p <- p + scale_x_continuous(position="top")
+  
+  info <- save_outputs(p, "axis_move_top")
+  expect_equal(length(info$data), 1)
+  expect_identical(info$layout$xaxis$side, "top")
+})
+
+test_that("Axis position moves to right", {
+  p <- p + scale_y_continuous(position="right")
+  
```

Comment:
The axis currently isn't shown because this PR hasn't yet addressed the issue of adding space for axis ticks/text in the right margin (and removing from the left) [as is done here](https://github.com/Alanocallaghan/plotly/blob/d3527db9b5fd5198d3b0752cfddfd53c6a486333/R/ggplotly.R#L640-L644)

Reply:
@cpsievert Is there any workaround to get the right side y axis to render in ggplotly when using `scale_y_continuous(position="right")` from ggplot → ggplotly?